doc/classes/Rect2.xml

```diff
@@ -4,10 +4,10 @@
 		A 2D axis-aligned bounding box using floating-point coordinates.
 	</brief_description>
 	<description>
-		[Rect2] consists of a position, a size, and several utility functions. It is typically used for fast overlap tests.
-		It uses floating-point coordinates. If you need integer coordinates, use [Rect2i] instead.
-		The 3D counterpart to [Rect2] is [AABB].
-		Negative values for [member size] are not supported and will not work for most methods. Use [method abs] to get a Rect2 with a positive size.
+		The [Rect2] built-in [Variant] type represents a rectangle in 2D space. It is defined by its [member position] and [member size], which are [Vector2]. Because it does not rotate, it is frequently used for fast overlap tests (see [method intersects]).
+		For integer coordinates, use [Rect2i]. The 3D equivalent to [Rect2] is [AABB].
+		[b]Note:[/b] Negative values for [member size] are not supported. With negative size, most [Rect2] methods do not work correctly. Use [method abs] to get an equivalent [Rect2] with a positive size.
+		[b]Note:[/b] In a boolean context, a [Rect2] evaluates to [code]false[/code] if it is empty (both [member position] and [member size] are [constant Vector2.ZERO]). Otherwise, a rectangle will always evaluate to [code]true[/code].
```

Comment:
The "_It always evaulates to true_" part is to be more consistent with all of the other Vector* descriptions. Although I do agree that using the future tense is weird.

Reply:
Oh, then I guess it makes sense like that. But "empty" as "position = zero and size = zero" seems strange (for me in this context an empty rectangle would be "size = zero; any position").

Reply:
Note that [up above](https://github.com/godotengine/godot/pull/69816#discussion_r1279845318) I've suggested an additional note (I'd guess you haven't noticed it since you haven't commented anything about it and it's not added). :upside_down_face: Opinions welcomed as always of course.

Reply:
Oh, I suspected that you're the one not noticing things in here! :smile: